src/frontends/pytorch/src/op/addmm.cpp

```diff
@@ -73,17 +73,9 @@ OutputVector translate_conv1d_ext(const NodeContext& context) {
     auto bias = context.get_input(2);
     bias = context.mark_node(std::make_shared<ov::op::v1::ConvertLike>(bias, x));
 
-    auto neg_one = context.mark_node(v0::Constant::create(element::i32, Shape{1}, {-1}));
-    auto zero = context.mark_node(v0::Constant::create(element::i32, Shape{1}, {0}));
-    auto shape_x = context.mark_node(std::make_shared<v3::ShapeOf>(x, element::i32));
-    auto x_last_dim = context.mark_node(std::make_shared<v8::Gather>(shape_x, neg_one, zero));
-    auto x_new_shape = context.mark_node(std::make_shared<v0::Concat>(OutputVector{neg_one, x_last_dim}, 0));
-
-    auto x_new = context.mark_node(std::make_shared<v1::Reshape>(x, x_new_shape, false));
-    auto mm = context.mark_node(std::make_shared<v0::MatMul>(x_new, weight));
+    auto mm = context.mark_node(std::make_shared<v0::MatMul>(x, weight));
     auto addmm = context.mark_node(std::make_shared<v1::Add>(bias, mm));
```

Comment:
Swapping `mm` and `bias` would allow us to skip this swapping on the plugin side.
```suggestion
    return {context.mark_node(std::make_shared<v1::Add>(mm, bias))};
```

Reply:
Fixed. Why do plugins do the swapping? Is it more efficient on their side?